app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
+  self.abstract_class = true
```

Comment:
This is just a convention in the version of Rails Accounts has and newer versions...
Models inherit from ApplicationRecord instead of ActiveRecord::Base so you can put common methods there.
I added it mostly because the generator `rails g model schools` already expected it to exist.